src/utilities/EventEmitter.test.ts

```diff
@@ -0,0 +1,242 @@
+import { it, describe, expect, beforeEach, afterEach, beforeAll } from 'vitest';
+import EventEmitter, { removeListener } from './EventEmitter';
+
+describe('Removes a listener from an array or object of listeners', () => {
```

Comment:
Perhaps we can just put here `removeListener` instead of trying to describe the interface?